cluster.tf

```diff
@@ -51,56 +69,49 @@ resource "google_container_cluster" "primary" {
   }
 
   addons_config {
+    # This is the default value but we want to be sure it will be always enabled
+    # This property will automatically create node pools based on resources consumption
+    horizontal_pod_autoscaling {
+      disabled = false
+    }
+
+    # Removed the to configure it as currently we don't want to deploy istio addon
     istio_config {
-      disabled = ! var.istio_config
+      disabled = true
     }
+
+    # CSI is a storage specification that allows cloud providers to provide vendor-specific support
+    # for storage features outside the k8s binary
+    gce_persistent_disk_csi_driver_config {
+      enabled = true
```

Comment:
Is this true by default?

Reply:
Reading about it, it seems the future way of connecting volumes.
https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/gce-pd-csi-driver
I'd go with it enabled